### PR TITLE
Fix checkstyle warnings

### PR DESCRIPTION
**Pull request by FlorisXll**
Resolved all Checkstyle warnings in the project.

### CODE REVIEW

File: src/main/java/nl/tudelft/pixelperfect/Scene.java [hunk 1]
```diff
@@ -13,7 +13,7 @@
 /**
  * Created by woute on 5/12/2016.
  *
- * Class for drawing objects in the game.
+ * <p>Class for drawing objects in the game.
```
Comment (jessetilro) [inline src/main/java/nl/tudelft/pixelperfect/Scene.java]: Fixed comments like these to contain an author tag in my branch.

**Comment by jessetilro** [comment]
LGTM

**Comment by dmalarev** [comment]
Looks good to me

File: src/test/java/nl/tudelft/pixelperfect/event/FireEventTest.java [hunk 2]
```diff
@@ -5,13 +5,14 @@
...
+   * Factory method for testing.
+   * 
+   * @return PlasmaLeakEvent class
+   */
+  @Override
+  public FireEvent createEvent() {
+    return new FireEvent(1, "TestEvent", "An Event to test the Class.", 42, 42, 99.42);
+  }
 
 }
```
Comment (wouterz) [inline src/test/java/nl/tudelft/pixelperfect/event/FireEventTest.java]: no newline, should be checkstyle error

Comment (FlorisXll) [inline src/test/java/nl/tudelft/pixelperfect/event/FireEventTest.java]: According to Checkstyle, everything is fine.

**Pull request merged**